docs/digger/installation.asciidoc

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+section: guides
+title: Digger installation guide
+toc_generate: true
+---
+
+== Digger macOS setup
+
```

Comment:
I think it will be a good idea to show some examples of the ansible command to show how to run the task to provision mac os servers